<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>664</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="customerButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Customer Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inventoryButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="text">
    <string>View Inventory</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>911</width>
     <height>811</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="customerPage">
    <widget class="QToolButton" name="customerPage_refreshButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ðŸ—˜</string>
     </property>
    </widget>
    <widget class="QTableView" name="customerPage_tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>562</width>
       <height>341</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID Number:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="customerPage_addButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>30</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Customer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RebateLineEdit">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>150</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Experation Date:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MemberTypeLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ExpirationDateLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total Spent:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rebate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Member Type:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TotalSpentLineEdit">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="customerPage_deleteButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>120</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete Customer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IDLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="inventoryPage">
    <widget class="QToolButton" name="inventoryPage_refreshButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ðŸ—˜</string>
     </property>
    </widget>
    <widget class="QTableView" name="InventoryTableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>671</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryPage_addButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>80</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Item</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryPage_editButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>20</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit Item</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>In Stock?</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ItemNameLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Item Price</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="QuantityLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Item Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="InStockLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Quantity</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryPage_deleteButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>140</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete Item</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ItemPriceLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Revenue</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RevenueLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="membershipPage">
    <widget class="QPushButton" name="membershipPage_convertButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Convert Memberships</string>
     </property>
    </widget>
    <widget class="QToolButton" name="membershipPage_refreshButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ðŸ—˜</string>
     </property>
    </widget>
    <widget class="QTableView" name="MembershipTableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>544</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="purchase">
    <widget class="QLineEdit" name="idLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="customerLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CustomerID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Item Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qtyLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>330</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="qtyLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>310</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantity</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="priceLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>390</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="priceLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>300</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Make Purchase</string>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>430</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dateLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>450</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="companyLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ðŸ’ª Bulk Club Inc.ðŸ’ª</string>
   </property>
  </widget>
  <widget class="QPushButton" name="membershipButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Memberships</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Test Purchase</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirmButton</sender>
   <signal>clicked()</signal>
   <receiver>Admin</receiver>
   <slot>test_purchase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>determineUpgradeOrDowngrade()</slot>
  <slot>test_purchase()</slot>
  <slot>test_page()</slot>
 </slots>
</ui>
